test/tests/h1.rs

```diff
@@ -34,7 +34,7 @@ async fn h1_get() -> Result<(), Error> {
         assert_eq!("GET Response", body);
     }
 
-    handle.try_handle()?.stop(false);
```

Comment:
before this change this would wait for the keep alive timeout, now it shut down instantly (because there is no pending request)